src/org/loklak/harvester/RedirectUnshortener.java

```diff
@@ -57,7 +57,7 @@
         "flip.it",
         "lnkd.in"
     };
```

Comment:
I guess it would be better, but I don't see the two arrays having any impact on the code (or why are they two different arrays in the first place). So juggling a service from one array to other won't make a difference I guess.